app/views/home/about.html.erb

```diff
@@ -143,7 +143,7 @@
   </div>
 
   <p>
-  Less than 1% of stories or comments get flagged, and users are not automatically punished by flags.
+  Fewer than 1% of stories or comments get flagged, and users are not automatically punished by flags.
```

Comment:
I draw a reviewer's attention to la wik's discussion of this topic:

https://en.wikipedia.org/wiki/Fewer_versus_less

Reply:
Yeah, pass on this one, it doesn't read right.

Reply:
Undone.